pyomo/contrib/parmest/parmest.py

```diff
@@ -31,7 +31,9 @@
 import pyomo.contrib.parmest.ipopt_solver_wrapper as ipopt_solver_wrapper
```

Comment:
I don't really know enough about parmest to judge this part - If @DLWoodruff and/or @kaklise are fine with it, then I am fine with it.